pkg/ingester/transfer.go

```diff
@@ -352,9 +352,12 @@ func (i *Ingester) TransferOut(ctx context.Context) error {
 	if i.cfg.MaxTransferRetries <= 0 {
 		return fmt.Errorf("transfers disabled")
 	}
+	if i.cfg.MaxTransferRetriesBackOff < i.cfg.MinTransferRetriesBackOff {
```

Comment:
This is not the right place to validate the configuration. Please add a `Validate()` function to `ingester.Config`, and call it from the [`cortex.Validate()`](https://github.com/cortexproject/cortex/blob/master/pkg/cortex/cortex.go#L121). See how other `Validate()` function works, for example [`storage.Config.Validate()`](https://github.com/cortexproject/cortex/blob/master/pkg/chunk/storage/factory.go#L64).

Please also add a unit test on the `Validate()` func.